src/plugins/ExportKeras/ExportKeras.js

```diff
@@ -0,0 +1,73 @@
+/*globals define*/
+/*eslint-env node*/
+define([
+    'plugin/PluginConfig',
+    'text!./metadata.json',
+    'SimpleNodes/Constants',
+    'plugin/GenerateKeras/GenerateKeras/GenerateKeras'
+], function (
+    PluginConfig,
+    pluginMetadata,
+    SimpleConstants,
+    PluginBase) {
+    'use strict';
+
+    pluginMetadata = JSON.parse(pluginMetadata);
+    const { exec } = require('child_process');
+    function ExportKeras() {
+        PluginBase.call(this);
+        this.pluginMetadata = pluginMetadata;
+        this.outputFileName = '';
+    }
+
+    ExportKeras.metadata = pluginMetadata;
+
+    // Prototypical inheritance from GenerateKeras.
+    ExportKeras.prototype = Object.create(PluginBase.prototype);
+    ExportKeras.prototype.constructor = ExportKeras;
+
+    ExportKeras.prototype.createOutputFiles = async function(activeNode){
+        const files = PluginBase.prototype.createOutputFiles.call(this, activeNode);
+        if(!this.outputFileName){
```

Comment:
We can probably just name it based on the active node (`${activeNode.name}.json`) and remove this config option.

Reply:
This has been addressed in latest commit